internal/xds/rbac/converter.go

```diff
@@ -24,13 +24,17 @@ import (
 	v1xdsudpatypepb "github.com/cncf/xds/go/udpa/type/v1"
 	v3xdsxdstypepb "github.com/cncf/xds/go/xds/type/v3"
 	v3rbacpb "github.com/envoyproxy/go-control-plane/envoy/config/rbac/v3"
+	v3auditloggersstreampb "github.com/envoyproxy/go-control-plane/envoy/extensions/rbac/audit_loggers/stream/v3"
```

Comment:
we had a change recently which attempts at standardizing pb named imports, so that we can rename them appropriately at import time. #6238 

@dfawley: should we make a note of this and make changes to the google3 import script?

Reply:
I believe I have the typed structs named correctly

Do you have a suggested name change for `v3auditloggersstreampb`? It's a new import, but it roughly takes the same approach of the last 2 tokens before `v3` `ignored/stuff/audit_loggers/stream/v3` -> `v3auditloggersstreampb` similar to `ignored/stuff/xds/type/v3` -> `v3xdstypepb`

Reply:
Sure this SGTM. I was just pinging Doug to see what is the action item for the import script. Resolving this.